gips/test/sys/t_modis.py

```diff
@@ -28,8 +28,15 @@ def setup_modis_data(pytestconfig):
     outcome = envoy.run(cmd_str)
     logger.info("MODIS data download complete.")
     if outcome.status_code != 0:
-        raise RuntimeError("MODIS data setup via `gips_inventory` failed",
-                           outcome.std_out, outcome.std_err, outcome)
+        msg = ("MODIS data setup via `gips_inventory` technically failed, but this may be due to false"
```

Comment:
Since this ignoring of failure is being added for a specific reason, can you add a comment or message indicating that it should be modified once Applied-GeoSolutions/gips/issues/201 is closed?